### PR TITLE
feat(bcd): draft bcd 4 index and support matrix

### PR DESCRIPTION
**Pull request by uguy**
TODO:

- Cross module links need to be check (syntax etc.) :heavy_check_mark: 
- Create alias or create redirects for deleted pages :heavy_check_mark: 
- Make PRs on cloud and bonita doc for osbolete links to deleted pages in this PR :heavy_check_mark:

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
-The Bonita Continuous Delivery (BCD) add-on provides a solution to use _Continuous Delivery_ practice for your Bonita Living Application development and deployment.
+image::images/bcd_capabilities.png[Bonita Continuous Delivery Capabilities]
 
-BCD lets you build a Living Application repository and deploy the generated artifacts from the command line. Besides, a pre-configured standalone https://jenkins.io/[Jenkins CI] environment is also provided as an *integration sample* with BCD.
+== Basic concepts
 
-image::images/bcd_capabilities.png[Bonita Continuous Delivery Capabilities]
+*BCD* offer has evolved now into *two Java command-line interfaces (CLI)* which allow to `build` and `deploy` Bonita Living Applications. You will be able to download the latest versions of the CLIs from our xref:{bonitaDocVersion}@bonita::bonita-repository-access.adoc[Bonita Artifact Repository] and then integrate the execution in your developer and CI pipelines.
+
+IMPORTANT: BCD features do not require docker anyomore. Usage has been greatly simplified. The only remaining xref:requirements-and-compatibility.adoc[prerequisites] are Java and Maven installed on the host running the CLIs.
```
Comment (ghost) [inline modules/ROOT/pages/index.adoc]: typo "anyomore"

File: modules/ROOT/pages/builder.adoc [hunk 2]
```diff
@@ -0,0 +1,217 @@
+= How to build a App repository
+:description: Build an App repository into a ready to deploy archive
+
+This page describes how to build Bonita Application projects from the command line.
+
+IMPORTANT: First make sure your maven installation is configured to access xref:{bonitaDocVersion}@bonita::bonita-repository-access.adoc[Bonita Artifact Repository] !
```
Comment (ghost) [inline modules/ROOT/pages/builder.adoc]: Warning the link to the BAR doc do not work on the preview

File: modules/ROOT/pages/builder.adoc [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+It's possible to extract the configuration to check it and also override (merge) some parameters if needed (see link:#extract[extract] and link:#merge[merge] commands).
+
+NOTE: If no parameters are defined on any processes, the `.bconf` file is not created.
+
+For instance, with a project in a directory named `bonita-vacation-management-example`, the following files will be generated:
+
+* `bonita-vacation-management-example/target/bonita-vacation-management-example-<environment>-<timestamp>.zip`
+* `bonita-vacation-management-example/target/bonita-vacation-management-example-<environment>-<timestamp>.bconf`
+
+NOTE: Since BCD 3.0.0, process configurations are built into a separate *.bconf* artifact. This configuration artifact allows to separate process configuration (parameters) from application assets. +
```
Comment (ghost) [inline modules/ROOT/pages/builder.adoc]: Suggestion: merge both notes about .bconf

File: modules/ROOT/pages/requirements-and-compatibility.adoc [hunk 4]
```diff
@@ -1,46 +1,51 @@
 = Requirements and compatibility
+:description: BCD requirements and compatibility matrix
 
-This page describes which system requirements need to be met in order to use the Bonita Continuous Delivery Add-On. +
-You will also find information about BCD and Bonita version compatibility.
+This page describes which system requirements need to be met in order to use the Bonita Continuous Delivery CLIs. +
+You will also find information about BCD CLIs and Bonita version compatibility.
 
-NOTE: You must always upgrade to one of the latest two versions of the BCD Controller image. Check how to xref:upgrade_bcd.adoc[migrate from an earlier version of BCD]. Only the last two images are now available on quay.io, older images will be removed.
+NOTE: Docker has been removed from the stack, this is a big improvement in usage facility and features remains the same since previous BCD versions where delegating work to the java CLIs now directly available.
```
Comment (ghost) [inline modules/ROOT/pages/requirements-and-compatibility.adoc]: I would remove this note. No value added

File: modules/ROOT/pages/upgrade_bcd.adoc [hunk 5]
```diff
@@ -1,146 +1,17 @@
...
-Move the directories where you have cloned your repositories.
-
-For example :
+=== From a 3.6.x towards {bcdVersion}
 
-[source,bash,subs="+macros"]
-----
-$ cp -r bonita-continuous-delivery_2.1.0/bonita-vacation-management-example bonita-continuous-delivery_pass:a[{bcdVersion}]/bonita-vacation-management-example
-----
+. Remove all BCD docker images since they are useless now.
```
Comment (ghost) [inline modules/ROOT/pages/upgrade_bcd.adoc]: I would remove "since they are useless now"

File: modules/ROOT/pages/builder.adoc (hunk 2 above)
Comment (uguy) [reply modules/ROOT/pages/builder.adoc]: Can't make the preview links for all cross components as of today. But syntax for this link is verified.

Comment (ghost) [reply modules/ROOT/pages/builder.adoc]: Ok, if that is normal (I mean, expected), fine by me.

File: modules/ROOT/pages/builder.adoc [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+----
+mvn dependency:copy -Dartifact=com.bonitasoft.tools:bonita-la-builder:<bonita.runtime.version>:jar:exec -Dmdep.stripVersion -Dmdep.stripClassifier -DoutputDirectory=./
+----
+
+Then just use the CLI jar once downloaded :
+
+.Execution sample
+[source, bash]
+----
+java -jar bonita-la-builder.jar build -e production <path to the bonita project>
```
Comment (rbioteau) [inline modules/ROOT/pages/builder.adoc]:
```suggestion
java -jar bonita-la-builder.jar build <path to the bonita project> -e production
```

File: modules/ROOT/pages/builder.adoc [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+== Build command
+
+An App repository can be created using Bonita Studio, and it can be xref:{bonitaDocVersion}@bonita::workspaces-and-repositories.adoc#_use_a_shared_project[shared with Git], so first ensure you have your App repository available on your filesystem.
+
+The Bonita Application Builder CLI version must correspond to the *bonita version* of the runtime. For instance the `bonita-la-builder.jar` in version `8.0.0` must be used to build artifacts meant to be deployed on a Bonita `8.0.0` runtime.
+
+[source, bash]
+----
+cd <project parent dir>
+java -jar bonita-la-builder.jar build -p <project dir> [options ...]
```
Comment (rbioteau) [inline modules/ROOT/pages/builder.adoc]:
```suggestion
java -jar bonita-la-builder.jar build <project dir> [options ...]
```

**Pull request merged**